### DISCUSSION TITLE
Delete Options: AEN, AEO, ON, OO, HARD ?!?

### DISCUSSION THREAD

**Discussion by Mae72DE** [qarmin/czkawka#1015] [Q&A]
Where can i find the Answere to these Methods:

  -D, --delete-method <DELETE_METHOD>
   Delete method (AEN, AEO, ON, OO, HARD) [default: NONE]

I think ChaptCPT just tells me just bulshit like this:
(So I may need the right Answer right from the Autor of the Programm)

   · AEN (AllEmptyNeighbour):
     Diese Methode löscht leere Dateien und Verzeichnisse
     und entfernt dabei auch alle leeren Nachbarn
     (Verzeichnisse ohne Unterordner oder Dateien).
     
   · AEN (Always Empty in Normal): 
     Diese Methode leert den Papierkorb und löscht die Dateien so, 
     dass sie nicht wiederhergestellt werden können. 
     Es überschreibt die Dateien mit zufälligen Daten 
     und wiederholt diesen Vorgang mehrmals.
     
   · AEO (AllEmptyOnly):
     Diese Methode löscht nur leere Dateien und Verzeichnisse,
     ohne die leeren Nachbarn zu entfernen.
     
   · AEO (Always Empty in Once): 
     Diese Methode leert den Papierkorb 
     und überschreibt die Dateien nur einmal mit zufälligen Daten. 
     Es ist weniger sicher als die AEN-Methode, 
     da die Daten möglicherweise wiederhergestellt werden können.

   · ON (OnlyNeighbour):
     Mit dieser Methode werden nur leere Nachbarn
     (Verzeichnisse ohne Unterordner oder Dateien) gelöscht,
     ohne die leeren Dateien und Verzeichnisse zu entfernen.
     
   · ON (Overwrite with Null):
     Diese Methode überschreibt die Dateien mit Nullen 
     und löscht sie anschließend. Es ist eine schnelle Methode, 
     aber weniger sicher, da Daten möglicherweise 
     wiederhergestellt werden können.

   · OO (OnlyOne):
     Diese Methode löscht nur eine Instanz
     jeder Datei oder jedes Verzeichnisses,
     wobei mindestens eine Instanz übrig bleibt,
     auch wenn sie leer ist.
     
     OO (Overwrite with Random Data Once): 
     Diese Methode überschreibt die Dateien nur einmal mit zufälligen Daten
     und löscht sie anschließend. Es ist sicherer als die ON-Methode, 
     aber immer noch weniger sicher als die AEN-Methode.

   · HARD:
     Diese Methode führt ein hartes Löschen durch,
     bei dem die Dateien und Verzeichnisse
     nicht in den Papierkorb verschoben werden,
     sondern endgültig gelöscht werden.
     
     HARD (Secure Deletion): 
     Diese Methode verwendet eine sichere Löschmethode wie "shred" (Linux) 
     oder "sdelete" (Windows), um die Dateien mehrmals zu überschreiben 
     und sicher zu löschen. 
     Es ist die sicherste Methode, aber auch die zeitaufwändigste.

   · NONE:
     Dies ist der Standardwert und bedeutet,
     dass keine spezifische Löschmethode ausgewählt ist.
     In diesem Fall werden keine Dateien oder Verzeichnisse gelöscht.

**Comment by notgudman** [comment]
Don't know if you're still looking for the answer, but I had the same question and found the answer in another thread: https://github.com/qarmin/czkawka/issues/1206#issuecomment-1957611988_

> -D, --delete-method <DELETE_METHOD>
>           Methods to delete the files.
>           AEN - All files except the newest,
>           AEO - All files except the oldest,
>           ON - Only 1 file, the newest,
>           OO - Only 1 file, the oldest
>           HARD - create hard link
>           NONE - not delete files

as per-usual, chatgpt just completely made something up 😂 

(edit) : I just realized this question was from June of _last year_ and not this year, I hope you were already able to have your question answered before now
